NVDSuppressions.xml

```diff
@@ -19,4 +19,23 @@
    <packageUrl regex="true">^pkg:npm/ansi\-html@.*$</packageUrl>
    <vulnerabilityName>1002522</vulnerabilityName>
 </suppress>
+<suppress>
+   <notes><![CDATA[
+   file name: ansi-html:0.0.7
+   Vunerability is if an attacker provides a malicious string, it will get stuck processing the input for an extremely long time.
+   For now there is no patch available for this vulnerability.
+   ]]></notes>
+   <packageUrl regex="true">^pkg:npm/ansi\-html@.*$</packageUrl>
+   <vulnerabilityName>1005059</vulnerabilityName>
+</suppress>
+<suppress>
+   <notes><![CDATA[
+   file name: json-schema:0.2.3
+   json-schema is vulnerable to Improperly Controlled Modification of Object Prototype Attributes ('Prototype Pollution')
```

Comment:
do we want to mention here that this vulnerable package is not a risk because it's not part of the deployed code?